.github/workflows/integration-test.yaml

```diff
@@ -104,68 +92,109 @@ jobs:
       - name: Dotnet build
         # limit cpu count for self-hosted due to memory constraints
         run: dotnet build ${{ inputs.runs-on == 'self-hosted' && '-maxcpucount:2' || '' }}
-      # TODO: Set up caching for /home/runner/.cache/ms-playwright so this doesn't take as long
-      # TODO: Ditto for /home/runner/.pnpm-store/v3
-      # TODO: Ditto for /home/runner/.nuget/packages
-      # TODO: Ditto for /var/cache/apt/archives/
-      - name: Playwright setup
-        working-directory: frontend
-        if: ${{ inputs.run-playwright }}
-        run: pnpm install && pnpm pretest
-      - name: Set up Playwright dependencies
-        working-directory: frontend
-        if: ${{ inputs.run-playwright && inputs.runs-on == 'self-hosted' }}
-        run: sudo pnpm exec playwright install-deps
       - name: Build for tests
         run: "dotnet build /p:MercurialVersion=$MERCURIAL_VERSION"
         env:
           MERCURIAL_VERSION: ${{ inputs.hg-version }}
 
-      - name: Integration tests (Playwright)
-        if: ${{ inputs.run-playwright }}
+      - name: Integration tests (.NET)
+        if: ${{ matrix.test == 'dotnet' }}
```

Comment:
I think this should be gone now?
Although, it would be handy to be able to run just playwright tests. I don't think we can do that currently 🤔.
Maybe just add an if to the job and one more optional parameter that defaults to true?
```suggestion
```

Reply:
Missed that one while editing; thanks. Fixed in commit 3c0ba1f7.

Also, the run-dotnet suggestion is a good one. I was going to remove the run-playwright input, but it's actually handier to have both run-dotnet and run-playwright so it's easy to pick and choose. Implemented in commit ae43cc9f.

Reply:
👍 For some reason it was really gratifying triggering [this action](https://github.com/sillsdev/languageforge-lexbox/actions/runs/9496572076) and expecting/hoping it wouldn't do anything. And it didn't 🎉 😆